### PR TITLE
feat: add docker enviroment

### PR DESCRIPTION
**Pull request by Trusted97**
In this PR i've dockerized the library with two container. 
One with PHP 7.1 and one with PHP 8.0

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+You can then build the images:
+
+```console
+docker compose up --build -d
+```
+
+Now you can run commands needed to work on the project. For example, say you want to install the dependencies on PHP 8.0:
+
+```console
+docker-compose run codice-fiscale-php-71 composer install
```
Comment (DavidePastore) [inline CONTRIBUTING.md]: This one should be `codice-fiscale-80`, right? 😄

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+```
+
+or for enter the container
+
+```console
+docker-compose exec codice-fiscale-php-71 sh
```
Comment (DavidePastore) [inline CONTRIBUTING.md]: The same, here 😉

**Review by DavidePastore — CHANGES_REQUESTED** [review]

**Comment by DavidePastore** [comment]
Thanks for your PR, @Trusted97! It took a bit but in the end here we are 😉 I hope you are well!

File: CONTRIBUTING.md (hunk 1 above)
Comment (Trusted97) [reply CONTRIBUTING.md]: I've fixed!

File: CONTRIBUTING.md (hunk 2 above)
Comment (Trusted97) [reply CONTRIBUTING.md]: Fixed here too

**Comment by DavidePastore** [comment]
Thanks for your contribution, @Trusted97!

**Pull request merged**